README.md

```diff
@@ -74,6 +74,43 @@ sourceSets {
 eclipseClasspath.dependsOn("compileJavacc")
 ```
 
+### Intellij
+
+If you are using Intellij and would like your gradle project to compile nicely in intellij and have the generated code in the build path; 
+you can add the `idea` plugin and add the generated path to the main sourceSet, and the `idea.module`.
+(The following uses the Kotlin DSL for gradle.)
+```gradle
```

Comment:
Just realized that this code block should be surrounded by blank lines: https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines